frontend/src/metabase/css/core/hide.css

```diff
@@ -49,30 +52,34 @@
     display: none !important;
   }
 }
+
 @media screen and (--breakpoint-min-md) {
   .md-show {
     display: inherit !important;
   }
 }
+
 /* large */
 
 @media screen and (--breakpoint-min-lg) {
   .lg-hide {
     display: none !important;
   }
 }
+
 @media screen and (--breakpoint-min-lg) {
   .lg-show {
     display: inherit !important;
   }
 }
 
 /* xl */
-@media screen and (--breakpoint-min-xl) h {
+@media screen and (--breakpoint-min-xl) {
```

Comment:
This line prompts me to search for all the util usage from this file. Apparently, only `sm-show` of the utils here is being used.

What do you think about removing other utils so there's no confusion that they might still be in use?